### PR TITLE
 dask ml packge supprt for travis build

### PR DESCRIPTION
**Pull request by asellappen**

### CODE REVIEW

File: travis-ymls/dask-ml.travis.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+dist: focal
+
+matrix:
+  include:
+  - env: 
+     - PYTHON_VERSION=3.6
+     - envFile='ci/environment-3.6.yaml'
+  - env: $PYTHON_VERSION=3.7
+     - PYTHON_VERSION=3.7
```
Comment (gururajrkatti) [inline travis-ymls/dask-ml.travis.yml:22]: Are both these variables (with and without $) required ?

File: travis-ymls/dask-ml.travis.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  - conda install -c power  scikit-learn
+  - pip install dask_glm --no-cache-dir
+  - conda env create --quiet --file=$envFile --name=dask-ml-test && conda list -n dask-ml-test
+   
+install:
+  -  python -m pip install --quiet --no-deps -e .
+
+script: 
+   #- not supported are ignored and the same failures are in pipeline ci tests,few python package are not support to ppc64le ,hence ignored.
+   -  python -m pytest -v tests -n10 --ignore=tests/model_selection/test_hyperband.py --ignore=dask/array/slicing.py --ignore=tests/test_kmeans.py --ignore=tests/model_selection/test_incremental_warns.py  --ignore=tests/test_spectral_clustering.py  --ignore=tests/model_selection/test_incremental.py;
```
Comment (gururajrkatti) [inline travis-ymls/dask-ml.travis.yml:75]: Are all these need to be ignored for intel as well ?

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

File: travis-ymls/dask-ml.travis.yml (hunk 1 above)
Comment (asellappen) [reply travis-ymls/dask-ml.travis.yml:22]: no ,not required. corrected now.

File: travis-ymls/dask-ml.travis.yml (hunk 2 above)
Comment (asellappen) [reply travis-ymls/dask-ml.travis.yml:75]: yes,its failing in pipeline amd64 ci tests as well .So ignored for both env.

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**